public/app/features/alerting/unified/components/rule-editor/DashboardPicker.tsx

```diff
@@ -190,8 +178,8 @@ export const DashboardPicker = ({ dashboardUid, panelId, isOpen, onChange, onDis
       {!selectedDashboardIsInPageResult && dashboardUid && (
```

Comment:
If we add `dasboardModel` as a condition here we could omit the optional chaining